src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -1300,7 +1294,9 @@ def merge_units(
             reloading waveforms if needed
         sparsity_overlap : float, default 0.75
             The percentage of overlap that units should share in order to accept merges. If this criteria is not
-            achieved, soft merging will not be possible and an error will be raised
+            achieved for a pair of units, soft merging will not be applied to them.
+        error_if_overlap_fails : bool, default: False
```

Comment:
- rename to `raise_error_if...`
- propagate to `curation` module
- propagate to `SortingAnalyzer.merge_units`